code/sha3/Hacl.SHA3.fst

```diff
@@ -148,3 +150,29 @@ let cshake256_frodo input_len input cstm output_len output =
   absorb s 136ul input_len input (byte 0x04);
   squeeze s 136ul output_len output;
   pop_frame ()
+
+[@ (Comment "Keccak instantiation with the rate equal to 1088 bits ")]
+val keccak_1088:
```

Comment:
IIRC Tezos is using exactly this version of keccak together with SHAs. In this case, the only modification they need to do is to use this function instead of keccak (and remove the rate). The extracted keccak function has a different API - it returns bool instead. So, it was just a way to simplify their life adapting the changes :)